src/ralph/conf.py

```diff
@@ -319,6 +319,8 @@ class Config(BaseSettingsConfig):
         },
     }
     PARSERS: ParserSettings = ParserSettings()
+    RUNSERVER_AUTH_OIDC_AUDIENCE: str = None
+    RUNSERVER_AUTH_OIDC_ISSUER_URI: AnyHttpUrl = None
```

Comment:
Maybe we should be more explicit and define a `RUNSERVER_AUTH_BACKEND` configuration variable? And thus list `RUNSERVER_AUTH_BACKENDS` as an Enum?

```suggestion
    RUNSERVER_AUTH_BACKEND = RUNSERVER_AUTH_BACKENDS.BASIC
    RUNSERVER_AUTH_OIDC_AUDIENCE: str = None
    RUNSERVER_AUTH_OIDC_ISSUER_URI: AnyHttpUrl = None
```

Reply:
It is more clear indeed!